client/blocks/importer/wordpress/import-everything/migration-success/index.tsx

```diff
@@ -0,0 +1,128 @@
+		( e: React.MouseEvent< HTMLButtonElement > ): void => {
+			e.preventDefault();
+			isWpAdmin
+				? recordTracksEvent( 'calypso_site_importer_navigate_wp_admin' )
+				: recordTracksEvent( 'calypso_site_importer_navigate_site' );
+			redirect( url, isWpAdmin );
+		};
+
+	return (
+		<>
+			<Title> { translate( 'Say hello to your new home' ) } </Title>
+			<SubTitle>{ translate( 'All set! Your content was successfully imported.' ) }</SubTitle>
+
+			<Card size="small">
+				<CardBody>
+					<HStack spacing={ 3 } justify="space-between">
+						<Flex
+							className="migration-success__ctas"
+							direction={ [ 'column', 'row' ] }
+							expanded={ true }
+							align="center"
+						>
+							<FlexItem>
+								<p className="migration-success__site">{ targetSite.domain }</p>
+							</FlexItem>
+
+							<FlexItem className="migration-success-ctas__buttons">
+								<Flex gap={ 4 }>
+									<Button variant="primary" onClick={ navigateToSite( targetSite.URL ) }>
+										{ translate( 'View site' ) }
+									</Button>
+									<Button variant="secondary" onClick={ navigateToSite( targetSite.URL, true ) }>
+										{ translate( 'Go to WP Admin' ) }
+									</Button>
+								</Flex>
+							</FlexItem>
+						</Flex>
+					</HStack>
+				</CardBody>
+			</Card>
+
+			<div className="migration-success-links-wrapper">
+				<VStack
+					spacing="1"
+					className="screen-confirmation__list-item-wrapper"
+					justify="space-between"
+				>
+					<strong className="screen-confirmation__list-item-title">
+						{ translate( 'Manage your site from anywhere' ) }
+					</strong>
+					<p className="screen-confirmation__list-item-description">
+						{ translate(
+							'The Jetpack mobile app for iOS and Android makes managing your email, domain, and website even simpler.'
+						) }
+					</p>
+					<Link
+						className="migration-success-links-wrapper__link"
+						to={ localizeUrl( 'https://apps.wordpress.com/mobile/' ) }
+						target="_blank"
+						rel="noopener noreferrer"
+						title={ translate( 'Get the app' ) }
+					>
+						{ translate( 'Get the app' ) }
+					</Link>
+				</VStack>
+
+				<VStack
+					spacing="1"
+					className="screen-confirmation__list-item-wrapper"
+					justify="space-between"
+				>
+					<strong className="screen-confirmation__list-item-title">
+						{ translate( 'Migration questions? find answers' ) }
+					</strong>
+					<p className="screen-confirmation__list-item-description">
+						{ translate(
+							'Explore our comprehensive support guides and find solutions to all your email inquiries.'
+						) }
+					</p>
+					<Link
+						className="migration-success-links-wrapper__link"
+						to={ localizeUrl(
+							'https://wordpress.com/support/import/import-a-sites-content/#after-importing'
+						) }
+						target="_blank"
+						rel="noopener noreferrer"
+						title={ translate( 'Migration support resources' ) }
+					>
+						{ translate( 'Migration support resources' ) }
```

Comment:
We don't have this translation yet. Let's add a simple `hasTranslation || isEnglishLocale` structure here, you can find examples around.

Reply:
As a temp thing we could use `Explore support resources` when not English locale

Reply:
Personally, since this PR is not urgent (maybe I am wrong here) I would add string freeze until we have the texts translated. I am not a fan of adding temporary code, the chances of forgetting to update it back is considerable. If this is important, then it makes sense, and otherwise, we can wait. WDYT?

Reply:
I'd apply that structure so we can ship this today. This changes the completion step and will probably conflict with the User step we want to add, so better to address all now.

Reply:
I added a note in the backlog to remember this: https://github.com/Automattic/wp-calypso/projects/122#card-91953240